content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -642,6 +642,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
   immutable for better safety and performance.
 - `IndexedJob`: Allows the [Job](/docs/concepts/workloads/controllers/job/)
   controller to manage Pod completions per completion index.
+- `IngressClassNamespacedParams`: Allow namespace-scoped parameters reference in
```

Comment:
This would also need to be added in the [table](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-alpha-or-beta-features) above